src/sfizz/Region.cpp

```diff
@@ -1324,3 +1546,22 @@ float sfz::Region::getBendInCents(float bend) const noexcept
 {
     return bend > 0.0f ? bend * static_cast<float>(bendUp) : -bend * static_cast<float>(bendDown);
 }
+
+sfz::Region::Connection& sfz::Region::getOrCreateConnection(const ModKey& source, const ModKey& target)
+{
+    auto pred = [&source, &target](const Connection& c)
+    {
+        return c.source == source && c.target == target;
+    };
+
+    auto it = std::find_if(connections.begin(), connections.end(), pred);
+    if (it != connections.end())
+        return *it;
+
+    sfz::Region::Connection c;
+    c.source = source;
+    c.target = target;
+
+    connections.push_back(c);
+    return connections.back();
```

Comment:
Why don't we have this as a return value of push_back/emplace_back in C++11, I'll never understand...

Reply:
same